docs/DataWorkflows/DataCycle.md

```diff
@@ -0,0 +1,671 @@
+---
+layout: default
+title: Data Cycle
+parent: Data Workflows
+nav_order: 1
+has_toc: true
+---
+# The Data Cycle
+
+The data cycle includes the following steps: 
+1. Fetching raw data
+3. Curation of datasets
+2. Fetching the data onto the right cluster
+3. Unzipping the data (as needed)
+4. Processing the data!
+5. Sharing the data
+
+Each of these steps is detailed here: 
+
+# Fetching Your Data
```

Comment:
I propose including datalad here. That's how I grab OpenNeuro datasets.

Reply:
added